src/components/Request/NewRequestForm.tsx

```diff
@@ -139,23 +203,79 @@ const NewRequestForm = () => {
 
         <Grade name="grade" form={myForm} />
 
+        <OfficeSymbol name="officeSymbol" form={myForm} />
+
+        <Supervisor name="supervisor" form={myForm} />
+
         <MPCN name="mpcn" form={myForm} />
 
         <CPCN name="cpcn" form={myForm} />
 
         <FMS name="fms" form={myForm} />
 
-        <OfficeSymbol name="officeSymbol" form={myForm} />
-
         <PositionSensitivity name="positionSensitivity" form={myForm} />
 
         <DutyLocation name="dutyLocation" form={myForm} />
```

Comment:
This sprint calls it Installation -- and says to include City and State -- should this component be updated to add in a helper text to specify City and State?